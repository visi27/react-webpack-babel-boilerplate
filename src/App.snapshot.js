import React from 'react';
import renderer from 'react-test-renderer';

import App, { Counter } from './App';

describe('App Snapshot', () => {
  test('renders', () => {
    const component = renderer.create(<App/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot('APP');
  });
});

describe('Counter Snapshot', () => {
  test('renders', () => {
    const component = renderer.create(<Counter counter={10}/>);
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot('COUNTER');
  });
});