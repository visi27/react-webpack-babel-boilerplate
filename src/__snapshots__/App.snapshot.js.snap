// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App Snapshot renders: APP 1`] = `
<div>
  <h1>
    My Counter
  </h1>
  <p>
    0
  </p>
  <button
    className="increment"
    onClick={[Function]}
    type="button"
  >
    Increment
  </button>
  <button
    className="decrement"
    onClick={[Function]}
    type="button"
  >
    Decrement
  </button>
</div>
`;

exports[`Counter Snapshot renders: COUNTER 1`] = `
<p>
  10
</p>
`;
